Fix some typos in building assistants tutorial

edmacovaz opened the pull request:
**Proposed changes**:
- Some very small fixes to typos to try and get the hang of things in here.

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

claassistantio commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/RasaHQ/rasa?pullRequest=5175) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/RasaHQ/rasa?pullRequest=5175) before we can accept your contribution.<br/><hr/>**Ed** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/RasaHQ/rasa?pullRequest=5175) it.</sub>

erohmensing commented:
@edmacovaz also please sign the CLA! 🙂

edmacovaz commented:
I think I have signed the CLA now. Also updated my git config and added my Rasa email to my Github account. Hopefully that connects things up properly...
